src/sbvr-api/sbvr-utils.d.ts

```diff
@@ -30,17 +30,27 @@ export type AnyObject = {
 	[key: string]: any;
 }
 
-export interface User {
+interface Actor {
+	permissions?: string[];
+}
+
+export interface User extends Actor {
 	id: number;
 	permissions?: string[];
```

Comment:
I guess that this can now be removed, since it is inherited from Actor now.

Reply:
Yeah it can be, I meant to but I forgot - removed it now